recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "git-annex" %}
-{% set version = "7.20200219" %}
+{% set version = "7.20200219.a0" %}
```

Comment:
but that would take precedence over the normal build, right? if you just want it for debugging purposes, you might like to establish a new `channel_target` (other than default `main`) and then it could be installable  from it.  For an example see our past "rc" target for datalad-feedstock:

<details>
<summary>diff from master... - majority is just rerendered (crucial portion is in recipe/conda_build_config.yaml iirc). </summary> 

```shell
diff --git a/.ci_support/linux_python3.6.yaml b/.ci_support/linux_python3.6.yaml
index 6b98523..6d2d20c 100644
--- a/.ci_support/linux_python3.6.yaml
+++ b/.ci_support/linux_python3.6.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 docker_image:
 - condaforge/linux-anvil-comp7
 pin_run_as_build:
diff --git a/.ci_support/linux_python3.7.yaml b/.ci_support/linux_python3.7.yaml
index acd1367..f9e25a8 100644
--- a/.ci_support/linux_python3.7.yaml
+++ b/.ci_support/linux_python3.7.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 docker_image:
 - condaforge/linux-anvil-comp7
 pin_run_as_build:
diff --git a/.ci_support/osx_python3.6.yaml b/.ci_support/osx_python3.6.yaml
index 13a9681..1c46076 100644
--- a/.ci_support/osx_python3.6.yaml
+++ b/.ci_support/osx_python3.6.yaml
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 macos_machine:
 - x86_64-apple-darwin13.4.0
 macos_min_version:
diff --git a/.ci_support/osx_python3.7.yaml b/.ci_support/osx_python3.7.yaml
index 91ac285..6b62650 100644
--- a/.ci_support/osx_python3.7.yaml
+++ b/.ci_support/osx_python3.7.yaml
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 macos_machine:
 - x86_64-apple-darwin13.4.0
 macos_min_version:
diff --git a/.ci_support/win_python3.6.yaml b/.ci_support/win_python3.6.yaml
index 3032431..0584b1a 100644
--- a/.ci_support/win_python3.6.yaml
+++ b/.ci_support/win_python3.6.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 pin_run_as_build:
   python:
     min_pin: x.x
diff --git a/.ci_support/win_python3.7.yaml b/.ci_support/win_python3.7.yaml
index 6eac991..36bbaa5 100644
--- a/.ci_support/win_python3.7.yaml
+++ b/.ci_support/win_python3.7.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 pin_run_as_build:
   python:
     min_pin: x.x
diff --git a/recipe/0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch b/recipe/0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch
deleted file mode 100644
index 6ff5ec9..0000000
Binary files a/recipe/0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch and /dev/null differ
diff --git a/recipe/conda_build_config.yaml b/recipe/conda_build_config.yaml
new file mode 100644
index 0000000..6cb0475
--- /dev/null
+++ b/recipe/conda_build_config.yaml
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge rc
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index 539c4cc..5b22d50 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,6 +1,6 @@
 {% set name = "datalad" %}
-{% set version = "0.11.8" %}
-{% set sha256 = "8461106756033c8d87ee2e104860a19f4cc14274fa5814aeb7f52a21770926c4" %}
+{% set version = "0.12.0rc6" %}
+{% set sha256 = "15f69017716954b97478a0188e94ca7f438fa16a2accdb4713c16e5425a98590" %}
 
 package:
   name: {{ name|lower }}
@@ -10,9 +10,6 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    # Should stay here
-    - 0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch
 
 build:
   number: 0
@@ -54,7 +51,6 @@ requirements:
     - pyld <0.8|>=0.8.2
     - pyperclip
     - pyyaml
-    - backports.lzma  # [py<33]
     - pillow
     - python
     - python-dateutil

```
</details>

and `conda install -c conda-forge/label/rc datalad` could be used to install it.  So you might like to establish "debug" channel and put custom builds there

Reply:
@yarikoptic an alpha version(.a0) would not take precedence over the normal build by default, unless you explicitly specify that version, if I understand correctly.  But thanks for reminding me about labels, it's the better way to do this.  I wish conda had something like MacPorts' "build variants", but conda-forge does not seem to support that...

Reply:
@yarikoptic So, do you then merge this into the master branch of the feedstock, temporarily overwriting the master with a version that uploads only to the rc label?

Reply:
I thought that it was picked up from a branch... In https://github.com/conda-forge/datalad-feedstock/pull/21 pr was merged into rc, not master branch

Reply:
@yarikoptic Do you by chance remember what you did to create the rc label in datalad, beyond creating the rc branch in the feedstock?  I've created the rc branch in the git-annex feedstock, but there is still no rc label in https://anaconda.org/conda-forge/git-annex/labels

Reply:
it seems that you didn't re-render after adding all the rc channel labeling.
<details>
<summary>diff on this feedstock</summary> 

```shell
$> git diff origin/master...origin/rc
diff --git a/conda_build_config.yaml b/conda_build_config.yaml
new file mode 100644
index 0000000..6cb0475
--- /dev/null
+++ b/conda_build_config.yaml
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge rc
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index 605067c..736b528 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -38,7 +38,7 @@ source:
     folder: ghc_src
 
 build:
-  number: 2
+  number: 3
   skip: True  # [not linux]
 
 requirements:
@@ -72,7 +72,7 @@ requirements:
     - rsync
   run:
     - rsync
-    - git >=2
+    - git
     - curl
     - openssh
     - lsof

```
</details>

<details>
<summary>diff on datalad feedstock</summary> 

```shell
diff --git a/.ci_support/linux_python3.6.yaml b/.ci_support/linux_python3.6.yaml
index 6b98523..6d2d20c 100644
--- a/.ci_support/linux_python3.6.yaml
+++ b/.ci_support/linux_python3.6.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 docker_image:
 - condaforge/linux-anvil-comp7
 pin_run_as_build:
diff --git a/.ci_support/linux_python3.7.yaml b/.ci_support/linux_python3.7.yaml
index acd1367..f9e25a8 100644
--- a/.ci_support/linux_python3.7.yaml
+++ b/.ci_support/linux_python3.7.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 docker_image:
 - condaforge/linux-anvil-comp7
 pin_run_as_build:
diff --git a/.ci_support/osx_python3.6.yaml b/.ci_support/osx_python3.6.yaml
index 13a9681..1c46076 100644
--- a/.ci_support/osx_python3.6.yaml
+++ b/.ci_support/osx_python3.6.yaml
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 macos_machine:
 - x86_64-apple-darwin13.4.0
 macos_min_version:
diff --git a/.ci_support/osx_python3.7.yaml b/.ci_support/osx_python3.7.yaml
index 91ac285..6b62650 100644
--- a/.ci_support/osx_python3.7.yaml
+++ b/.ci_support/osx_python3.7.yaml
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 macos_machine:
 - x86_64-apple-darwin13.4.0
 macos_min_version:
diff --git a/.ci_support/win_python3.6.yaml b/.ci_support/win_python3.6.yaml
index 3032431..0584b1a 100644
--- a/.ci_support/win_python3.6.yaml
+++ b/.ci_support/win_python3.6.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 pin_run_as_build:
   python:
     min_pin: x.x
diff --git a/.ci_support/win_python3.7.yaml b/.ci_support/win_python3.7.yaml
index 6eac991..36bbaa5 100644
--- a/.ci_support/win_python3.7.yaml
+++ b/.ci_support/win_python3.7.yaml
@@ -1,7 +1,7 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc
 pin_run_as_build:
   python:
     min_pin: x.x
diff --git a/recipe/0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch b/recipe/0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch
deleted file mode 100644
index 6ff5ec9..0000000
Binary files a/recipe/0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch and /dev/null differ
diff --git a/recipe/conda_build_config.yaml b/recipe/conda_build_config.yaml
new file mode 100644
index 0000000..6cb0475
--- /dev/null
+++ b/recipe/conda_build_config.yaml
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge rc
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index 539c4cc..5b22d50 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,6 +1,6 @@
 {% set name = "datalad" %}
-{% set version = "0.11.8" %}
-{% set sha256 = "8461106756033c8d87ee2e104860a19f4cc14274fa5814aeb7f52a21770926c4" %}
+{% set version = "0.12.0rc6" %}
+{% set sha256 = "15f69017716954b97478a0188e94ca7f438fa16a2accdb4713c16e5425a98590" %}
 
 package:
   name: {{ name|lower }}
@@ -10,9 +10,6 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    # Should stay here
-    - 0001-Use-backports.lzma-instead-of-pyliblzma-in-setup.py.patch
 
 build:
   number: 0
@@ -54,7 +51,6 @@ requirements:
     - pyld <0.8|>=0.8.2
     - pyperclip
     - pyyaml
-    - backports.lzma  # [py<33]
     - pillow
     - python
     - python-dateutil
```
</details>

so you are missing changes to `.ci_support/`:
```
(git)lena:~datalad/distributions/git-annex-feedstock[rc]git
$> git grep conda-forge -- .ci_support/
.ci_support/linux_.yaml:- conda-forge,defaults
.ci_support/linux_.yaml:- conda-forge main
```
still lists "main".

And I am not 100% sure that name (i.e. `rc`) matters.  You might like dedicated `debug` or smth like that better ;)